pkgs/servers/portunus/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "portunus";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "majewsky";
+    repo = "portunus";
+    # rev = "v${version}";
```

Comment:
Stale code? Or maybe add a doc comment?

Reply:
Testing artifact, will go when we got 1.1.0 released.